custom_model_runner/requirements.txt

```diff
@@ -26,4 +26,4 @@ termcolor
 packaging
 markupsafe<=2.1.3
 pydantic==2.9.2
```

Comment:
@zohar-mizrahi do we have to pin `pydantic`

Reply:
No. I've noticed that you have a follow up PR to test it.